src/site/content/en/blog/sms-receiver-announce/index.md

```diff
@@ -0,0 +1,395 @@
+3. Select your browser channel from the provided list.
+4. Press **Copy** to copy the text message and send it to another phone.
+5. Press **Verify**.
+6. From the other phone, send yourself the copied text message via SMS.
+
+Did you receive the SMS and see the prompt to enter the code to the input area?
+That is how the SMS Receiver API works for end users.
+
+{% Aside 'warning' %}
+If you are using a Work Profile on your Android device and the SMS Receiver does
+not seem to be working, try installing and using Chrome on your personal profile
+instead (i.e. the same profile in which you receive SMS messages).
+{% endAside %}
+
+## Using the SMS Receiver API
+
+Using the SMS Receiver API consists of two parts: JavaScript code in your web
+app and the formatted message text sent via SMS.
+
+{% Aside %}
+The SMS Receiver API requires a secure origin (HTTPS).
+{% endAside %}
+
+### Feature detection
+
+Feature detection is much the same as for many other APIs:
+
+```js
+if ('sms' in navigator) {
+  ...
+}
+```
+
+### Process the OTP
+
+The SMS Receiver API itself is simple enough:
+
+```js
+const sms = await navigator.sms.receive();
+```
+
+Once a user taps **Verify**, displayed in the bottom sheet, the promise
+containing the entire text message will resolve. You can use a regular
+expression to extract the OTP and verify the user. Notably, you should parse and
+use the SMS message assuming it could have been altered by an attacker inserting
+their own SMSes into your app (e.g. following the formatting convention and
+sending it right after you called `navigator.sms.receive()`). For example, if a
+text message contains a six digit verification code following `otp=`, the code
+would look like this:
+
+```js
+const code = sms.content.match(/^[\s\S]*otp=([0-9]{6})[\s\S]*$/m)[1];
+```
+
+You can now submit the code to the server to verify it.
+
+### Formatting the SMS message
+
+The API itself should have looked simple enough, but a critical part is to
+format your SMS text message according to a specific convention. The message has
+to be sent after `navigator.sms.receive()` is called and must comply with a
+formatting convention.
+
+The SMS message must be received on the device where `navigator.sms.receive()`
+was called.
+
+The message must adhere to the following formatting:
+
+* The origin part of the URL of the website that invoked the API. It must be
+  preceded by `For: `.
+* The URL must contain (for [the time
+  being](https://github.com/samuelgoto/sms-receiver/issues/4#issuecomment-528991114))
+  a query parameter whose value is the application hash of the user's Chrome
+  instance. (These are static strings. See the table below.)
+* The URL must contain a query parameter `otp` whose value is the OTP.
+
+An example message that can be retrieved by the browser would look like this:
+
+```text
+Your OTP is: 123456.
+
+For: https://sms-receiver-demo.glitch.me/?otp=123456&PqEvUq15HeK
```

Comment:
Can we make sure the application hash matches one of the ones listed below (either Beta or Stable)?

Reply:
Very good catch! I just fixed it.